avm/res/sql/instance-pool/tests/e2e/waf-aligned/main.test.bicep

```
@@ -56,7 +62,8 @@ module testDeployment '../../../main.bicep' = {
   params: {
     name: nestedDependencies.outputs.sqlInstancePoolName
     location: enforcedLocation
-    skuName: 'GP_Gen8IM'
+    skuName: skuName
     subnetResourceId: nestedDependencies.outputs.subnetId
+    enableTelemetry: enableTelemetry
```

```suggestion
    skuName: 'GP_Gen5'
    subnetResourceId: nestedDependencies.outputs.subnetId
```

Should remain hardcoded here as it otherwise won't be pulled into the readme.
Also the `enableTelemetry` parameter is not required to be `waf-aligned` which is the point of this test.
Note that no telemetry will be collected by this test as deployments with the naming pattern `-..--..-` or rather `--------`  are ignored - which is the naming pattern of any non-published module.